### PR TITLE
Add badges and information for LTS and support

### PR DESCRIPTION
**Pull request by seabaylea**
### Description
Add badge and information for Module LTS for Loopback v3.x.x, and add a badge for IBM Frameworks support for Loopback.

#### Related issues

- N/A

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,4 +99,12 @@ Contributions to the LoopBack project are welcome! See [Contributing to LoopBack
 
 One of the easiest ways to contribute to LoopBack is to report an issue. See [Reporting issues](https://loopback.io/doc/en/contrib/Reporting-issues.html) for more information.
 
+## Module Long Term Support Policy
+
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Module Version   | Release Date | Minimum EOL | EOL With     | Status  |
+|------------------|--------------|-------------|--------------|---------|
+| V3.x.x	         | Sept 2016    | Dec 2019    |              | Current |
```
[the remark this answers is not in the document]
Comment (raymondfeng) [reply README.md]: I'm fine either way.

[the remark this answers is not in the document]
Comment (seabaylea) [reply README.md]: We can drop the 'v' and move to `3.x.x.` - it's probably the cleanest to read.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # LoopBack
 
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/strongloop/loopback?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Module LTS Adopted'](https://img.shields.io/badge/Module%20LTS-Adopted-brightgreen.svg?style=flat)](http://github.com/CloudNativeJS/ModuleLTS)
+[![IBM Support](https://img.shields.io/badge/Support-IBM%20Frameworks-brightgreen.svg?style=flat)](http://ibm.biz/node-support)
```
[the remark this answers is not in the document]
Comment (seabaylea) [reply README.md]: There's two available support levels: `IBM Runtimes` (which covers Node.js only) and `IBM Frameworks` which includes frameworks and modules. For Loopback you need to get the "Framework" level.

We could switch to "IBM Support: Frameworks" if that makes more sense.

**Comment by raymondfeng** [comment]
+1 for `IBM Support: Frameworks`.

**Comment by seabaylea** [comment]
@dhmlau @bajtos I think April 2019 is the likely target date for 2.x EOL anyway. Do you want me to update the table with that, or leave as-is?

**Comment by dhmlau** [comment]
Apologize that I didn't get the release date correctly.  In this case, I think the table should be: 

| Module Version   | Release Date | Minimum EOL | EOL With     | Status  |
|------------------|--------------|-------------|--------------|---------|
| 3.x.x	         | Dec 2016    | Dec 2019    |              | Current |
| 2.x.x	         | July 2014    | Apr 2019   | Node.js 6.x  | LTS     |

**Pull request merged**